crates/cheatnet/tests/cheatcodes/warp.rs

```diff
@@ -318,3 +319,144 @@ fn warp_library_call() {
     .unwrap();
     assert_outputs(before_warp_output, after_warp_cancellation_output);
 }
+
+#[test]
+fn warp_all_simple() {
```

Comment:
Please add a test with `multiple` as well
And an e2e test (or a few cases) with the actual cairo interface to test its' interface for errors/changes